CMakeLists.txt

```diff
@@ -31,3 +35,10 @@ if(ENABLE_TESTS)
   enable_testing()
   add_subdirectory(tests)
 endif()
+
+# Determine whether or not to generate documentation.
+OPTION(ENABLE_DOCS "Enable generation of doxygen-based documentation." OFF)
```

Comment:
Could you make the CMake lower-case as you did in wrf_io?

Reply:
Yes. I really need to do this in NCEPLIBS-ip, which is where I keep copying from...